cpu/msp430_common/periph/pm.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_msp430_common
+ * @{
+ *
+ * @file
+ * @brief       MSP430 common periph/pm functions
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+
+void pm_reboot(void)
+{
+    /* force an hardware reboot ("Power-Up Clear"), by writing
```

Comment:
Ups, this was not on purpose -> simply forgot to copy the old copyright into my template file... Sorry!